File: pkg/codegen/testing/test/testdata/simple-range-pp/go/simple-range.go
package main

import (
	"fmt"

	"github.com/pulumi/pulumi-aws/sdk/v4/go/aws/s3"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		var bucket []*s3.Bucket
		for index := 0; index < 10; index++ {
			key0 := index
			val0 := index
			__res, err := s3.NewBucket(ctx, fmt.Sprintf("bucket-%v", key0), &s3.BucketArgs{
				Website: &s3.BucketWebsiteArgs{
					IndexDocument: pulumi.String(fmt.Sprintf("index-%v.html", val0)),
				},
			})
			if err != nil {
				return err
			}
			bucket = append(bucket, __res)
		}
		return nil
	})
}

File: pkg/codegen/testing/test/testdata/simple-range-pp/python/simple-range.py
import pulumi
import pulumi_aws as aws

bucket = []
for range in [{"value": i} for i in range(0, 10)]:
    bucket.append(aws.s3.Bucket(f"bucket-{range['value']}", website=aws.s3.BucketWebsiteArgs(
        index_document=f"index-{range['value']}.html",
    )))
